org.js.nuclear.Nuclear.metainfo.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>org.js.nuclear.Nuclear</id>
+    <metadata_license>FSFAP</metadata_license>
+    <project_license>GPL-3.0+</project_license>
+    <name>nuclear music player</name>
+    <summary>A electron based music player</summary>
+    <content_rating type="oars-1.1" />
+    <description>
+    <p>nuclear is a free music streaming program that pulls content from free sources all over the internet.</p>
+
+    <p>If you know mps-youtube, this is a similar music player but with a GUI.
+    It's also focusing more on audio. Imagine Spotify which you don't have to pay for and with a bigger library.</p>
+
+    <p>Features</p>
+    <ul>
+        <li>- Searching for and playing music from YouTube (including integration with playlists), Jamendo, and SoundCloud</li>
```

Comment:
Since it's a `<li>` (list item) you shouldn't have a dash at the begining.

Reply:
okay, i will remove it.